docs/reference.md

```diff
@@ -333,6 +333,10 @@ $ curl -u USERNAME:PASSWORD https://openwhisk.ng.bluemix.net/api/v1/namespaces/w
 ]
 ```
 
+The API of OpenWhisk also supports asynchronous calls on websites from the client.
+To allow these calls from the browser directly, OpenWhisk responds to OPTIONS calls with Cross-Origin Resource Sharing headers for allowed origins and headers. Access-Control-Allow-Origin will always be * and Access-Control-Allow-Headers will always yield Authorization and Content-Type.
+
```

Comment:
Suggested rewording for the above:

The OpenWhisk API also supports request-response calls from web clients. OpenWhisk responds to `OPTIONS` requests with Cross-Origin Resource Sharing headers for allowed origins and headers. Currently, all origins are allowed (i.e., Access-Control-Allow-Origin  is `*`) and Access-Control-Allow-Headers yield Authorization and Content-Type.